setupJest.js

```diff
@@ -1,3 +1,4 @@
+'use strict'
```

Comment:
I think `'use strict';` is now supposed to be referenced in every file. Also would you mind adding a comma and extra newline?